app/src/main/java/com/rahobbs/todo/interfaces/ItemTouchHelperAdapter.java

```diff
@@ -0,0 +1,13 @@
+package com.rahobbs.todo.interfaces;
+
+
+/**
+ * Created by rachael on 3/4/16.
+ */
+public interface ItemTouchHelperAdapter {
```

Comment:
don't use the word adapter here. Adapter has other meanings in android so you want to avoid misnaming things for confusion!